app/finders/conversation_finder.rb

```diff
@@ -79,6 +79,8 @@ def find_all_conversations
     if params[:conversation_type] == 'mention'
       conversation_ids = current_account.mentions.where(user: current_user).pluck(:conversation_id)
       @conversations = current_account.conversations.where(id: conversation_ids)
+    elsif params[:conversation_type] == 'unattended'
```

Comment:
Showing **rubocop** warnings while running the app. This change in code will fix the issue.

```ruby
  def set_assignee_type
    @assignee_type = params[:assignee_type]
  end
```

```ruby
  def find_all_conversations
    case @conversation_type
    when 'mention'
      conversation_ids = current_account.mentions.where(user: current_user).pluck(:conversation_id)
      @conversations = current_account.conversations.where(id: conversation_ids)
    when 'unattended'
      @conversations = current_account.conversations.where(inbox_id: @inbox_ids, first_reply_created_at: nil)
    else
      @conversations = current_account.conversations.where(inbox_id: @inbox_ids)
    end
    @conversations
  end
```

Reply:
Thanks, I have added them.